src/EnergyPlus/RoomAirModelManager.cc

```diff
@@ -1350,6 +1365,463 @@ namespace RoomAirModelManager {
+		} // loop thru NumOfRoomAirflowNetControl
+
+		cCurrentModuleObject = "RoomAir:Node:AirflowNetwork";
+		TotNumOfRoomAFNNodes = GetNumObjectsFound( cCurrentModuleObject );
+		for ( Loop = 1; Loop <= TotNumOfRoomAFNNodes; ++Loop ) {
+			GetObjectItem( cCurrentModuleObject, Loop, cAlphaArgs, NumAlphas, rNumericArgs, NumNumbers, status, _,
+				lAlphaFieldBlanks, cAlphaFieldNames, cNumericFieldNames );
+			ZoneNum = FindItemInList( cAlphaArgs( 2 ), Zone.Name( ), NumOfZones );
+			if ( ZoneNum == 0 ) {
+				ShowSevereError( "GetRoomAirflowNetworkData: Invalid " + cAlphaFieldNames( 2 ) + " = " + cAlphaArgs( 2 ) );
+				ShowContinueError( "Entered in " + cCurrentModuleObject + " = " + cAlphaArgs( 1 ) );
+				ShowContinueError( "Not a valid zone name." );
+				ErrorsFound = true;
+				continue;
+			}
+
+			RAFNNodeNum = FindItemInList( cAlphaArgs( 1 ), RoomAirflowNetworkZoneInfo( ZoneNum ).Node.Name( ), RoomAirflowNetworkZoneInfo( ZoneNum ).NumOfAirNodes );
+			if ( RAFNNodeNum == 0 ) {
+				ShowSevereError( "GetRoomAirflowNetworkData: Invalid " + cAlphaFieldNames( 2 ) + " = " + cAlphaArgs( 2 ) );
+				ShowContinueError( "Entered in " + cCurrentModuleObject + " = " + cAlphaArgs( 1 ) );
+				ShowContinueError( "Not a valid RoomAir:Node:AirflowNetwork name." );
+				ErrorsFound = true;
+				continue;
+			}
+
+			RoomAirflowNetworkZoneInfo( ZoneNum ).Node( RAFNNodeNum ).ZoneVolumeFraction = rNumericArgs( 1 );
+			if ( !lAlphaFieldBlanks( 3 ) ) {
+				RoomAirflowNetworkZoneInfo( ZoneNum ).Node( RAFNNodeNum ).NodeSurfListName = cAlphaArgs( 3 );
+			}
+			else {
+				RoomAirflowNetworkZoneInfo( ZoneNum ).Node( RAFNNodeNum ).HasSurfacesAssigned = false;
+			}
+			if ( !lAlphaFieldBlanks( 4 ) ) {
+				RoomAirflowNetworkZoneInfo( ZoneNum ).Node( RAFNNodeNum ).NodeIntGainsListName = cAlphaArgs( 4 );
+			}
+			else {
+				RoomAirflowNetworkZoneInfo( ZoneNum ).Node( RAFNNodeNum ).HasIntGainsAssigned = false;
+			}
+			if ( !lAlphaFieldBlanks( 5 ) ) {
+				RoomAirflowNetworkZoneInfo( ZoneNum ).Node( RAFNNodeNum ).NodeHVACListName = cAlphaArgs( 5 );
+			}
+			else {
+				RoomAirflowNetworkZoneInfo( ZoneNum ).Node( RAFNNodeNum ).HasHVACAssigned = false;
+			}
+
+		} // loop thru TotNumOfRoomAFNNodes
+
+		cCurrentModuleObject = "RoomAir:Node:AirflowNetwork:AdjacentSurfaceList";
+		TotNumOfRAFNNodeSurfLists = GetNumObjectsFound( cCurrentModuleObject );
+		for ( Loop = 1; Loop <= TotNumOfRAFNNodeSurfLists; ++Loop ) {
+			foundList = false;
+			GetObjectItem( cCurrentModuleObject, Loop, cAlphaArgs, NumAlphas, rNumericArgs, NumNumbers, status, _, _, cAlphaFieldNames, cNumericFieldNames );
+			for ( ZoneNum = 1; ZoneNum <= NumOfZones; ++ZoneNum ) {
+				// find surface list
+				if ( RoomAirflowNetworkZoneInfo( ZoneNum ).NumOfAirNodes > 0 ) {
+					RAFNNodeNum = FindItemInList( cAlphaArgs( 1 ), RoomAirflowNetworkZoneInfo( ZoneNum ).Node.ma( &RoomAirflowNetworkAirNodeNestedStruct::NodeSurfListName ), RoomAirflowNetworkZoneInfo( ZoneNum ).NumOfAirNodes );
```

Comment:
@lgu1234 There should be a new version of FindItemInList that doesn't require calling ma directly, which we should avoid. I think this is it:

https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/InputProcessor.hh#L667